src/services/alb/data.ts

```diff
@@ -623,6 +723,34 @@ export default async ({
     })
 
     await Promise.all(targetHealthPromises)
+
+    /**
+     * Step 7, use the describeListenerCertificates method to get the certificates IDs
+     */
+    albData.forEach(alb => {
+      const { listeners, region } = alb
```

Comment:
albsData won't be null because we are initializing it in line 153.
Regarding listerners, you are right, we are no sure so I set a default value. Thanks for pointing it to me